app/process/roads_bikelanes/roads_bikelanes.lua

```diff
@@ -129,15 +129,9 @@ function osm2pgsql.process_way(object)
       local id = result.prefix .. ':' .. signSideMapping[result.sign] .. '/' .. object.id
 
       -- Hacky cleanup tags we don't need to make the file smaller
-      result._infrastructureExists = nil -- not used in atlas-app
       result.segregated = nil            -- no idea why that is present in the inspector frontend for way 9717355
-      result.sign = nil                  -- not used in atlas-app
-      result.side = nil                  -- not used in atlas-app
-      -- result.offset = nil                -- not used in atlas-app -- DANGER: This one is used in bikelanes.sql
-      result.parent = nil                -- not used in atlas-app
```

Comment:
Update und wieder rein

Reply:
hab das in die Funktion `ExtractPublicTags` ausgelagert